entries/w/woltlab.com.json

```diff
@@ -1,15 +1,16 @@
 {
   "WoltLab": {
     "domain": "woltlab.com",
+    "tfa": [
+      "totp",
+      "u2f",
+      "email"
+    ],
+    "documentation": "https://www.woltlab.com/account-security/",
```

Comment:
The documentation page can't 403 I'm afraid. Could you post a screenshot of the page here?

Reply:
The linked "documentation" is the page where the user is able to set-up multi-factor authentication and thus only visible to logged in users. We do not have a dedicated manual for this, because we try to make the process as self-explanatory as possible.

Find screenshots of the page of my local development environment below. The first screenshot is what the user will see when following the link.

![Screen Shot 2021-08-14 at 12 31 01](https://user-images.githubusercontent.com/209270/129443530-a6ed4b5a-ebda-4a44-8eaf-752ac876efdb.png)
![Screen Shot 2021-08-14 at 12 31 13](https://user-images.githubusercontent.com/209270/129443529-d35b692e-0588-4a37-8d31-39edc9093c1a.png)
![Screen Shot 2021-08-14 at 12 31 27](https://user-images.githubusercontent.com/209270/129443528-b7a74a02-49f2-49e6-b8a6-76537ad8bbcc.png)
![Screen Shot 2021-08-14 at 12 32 10](https://user-images.githubusercontent.com/209270/129443527-c37981f0-1133-48bf-b0cc-d3fd709e4442.png)
![Screen Shot 2021-08-14 at 12 34 11](https://user-images.githubusercontent.com/209270/129443526-f8db1a1a-a980-4927-b593-e453c17565c4.png)

If this is not what you expect to see for a documentation link, just let me know. I'll remove the link then.

Reply:
Thanks for the screenshots. As it is the actual setup page and not a documentation page it still needs to be removed though.
```suggestion
```

Reply:
Understood. Removed now.